contracts/examples/bonding-curve/src/utils/storage.rs

```diff
@@ -0,0 +1,27 @@
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+use crate::{
+	function_selector::CurveArguments, function_selector::FunctionSelector,
+	function_selector::Token,
+};
+
+#[elrond_wasm_derive::module]
+pub trait StorageModule {
+	#[view(lastErrorMessage)]
+	#[storage_mapper("last_error_message")]
+	fn last_error_message(&self) -> SingleValueMapper<Self::Storage, BoxedBytes>;
+
+	#[storage_mapper("bonding_curve")]
+	fn bonding_curve(
```

Comment:
As clippy suggest. The name suggest that the storage stores a bonding curve. but it is actually a tuple with 3 elements. Is the tuple the actual curve? If so, please consider creating a new struct called "BondingCurve" and have the 3 elements as struct members.